test/gtest/ucp/ucp_test.cc

```diff
@@ -113,6 +113,18 @@ bool ucp_test::has_transport(const std::string& tl_name) const {
     return check_transport(tl_name, GetParam().transports);
 }
 
+bool
```

Comment:
Try to keep the same order of methods in header and implementation filed.

Reply:
actually these changes can be reverted. (no need to refactor this function out of test_wireup)